sources/packages/api/src/types/userInfo.ts

```diff
@@ -3,4 +3,6 @@ export interface UserInfo {
   givenNames: string;
   email: string;
   userName: string;
+  birthdate: string;
```

Comment:
Because our model is organized differently, i just think it makes more sense to either introduce another studentInfo.ts with the additional fields instead of putting it all here in userInfo.ts
and
We can rename this userInfo.ts to call this as BaseUserInfo.ts

Reply:
If you look from a database perspective our models are in a way that does not make sense to add these properties here.
That said, my understanding was that this object was representing our business model, where both properties belong to the user and not the student. Let's talk about it.